cli/dts/lib.esnext.d.ts

```diff
@@ -22,4 +22,3 @@ and limitations under the License.
 /// <reference lib="esnext.intl" />
 /// <reference lib="esnext.string" />
 /// <reference lib="esnext.promise" />
-/// <reference lib="esnext.weakref" />
```

Comment:
This was a manual patch until it is available in TypeScript (see: #8056)... Please do not remove.

Reply:
reintrouduced in [`a9d6dae` (#8138)](https://github.com/denoland/deno/pull/8138/commits/a9d6dae2edc2c99d63a50ac9244f18b83e03e903)